modules/appeals_api/lib/appeals_api/token_validation_client.rb

```diff
@@ -4,28 +4,39 @@
 require 'token_validation/v2/configuration'
 
 module AppealsApi
-  class TokenValidation
-    module V3
-      class Configuration < ::TokenValidation::V2::Configuration
-        def connection
-          Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
-            conn.request :url_encoded # required for token validation service v3 (v2 accepted json)
-            conn.response :snakecase
-            conn.adapter Faraday.default_adapter
-          end
-        end
+  # Represents the result of token validation
+  #
+  # @attr [Array<String>] scopes the scopes included with this token
+  # @attr [String|nil] veteran_icn the ICN of the target veteran, if any
+  TokenValidationResult = Struct.new(:scopes, :veteran_icn)
```

Comment:
I made this struct to encapsulate any token validation information that could be useful outside of this client and make it available via the `token_validation_result` in the openid_auth.rb concern.